exasol/nb_connector/bfs_utils.py

```diff
@@ -10,15 +10,15 @@
 _logger = logging.getLogger(__name__)
 
 
-def _file_in_bucket(file_name: str, bucket: bfs.BucketLike) -> bool:
+def _file_in_bucket(file_name: str, bucket: bfs.Bucket) -> bool:
```

Comment:
I understand this is due to other functions that declare bucket as `bfs.Bucket`. But I think it is wrong there. Ideally we should be using `bfs.BucketLike` everywhere. So, depending on what we want to do now, we can do a quick fix like you did here, or do it properly and change Bucket -> BucketLike elsewhere.